python/llm/dev/benchmark/all-in-one/run-spr.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
-
+source bigdl-nano-init -c
+export OMP_NUM_THREADS=48
+export TRANSFORMERS_OFFLINE=1
 
 # set following parameters according to the actual specs of the test machine
 numactl -C 0-47 -m 0 python $(dirname "$0")/run.py
```

Comment:
maybe accept core# as a parameter to automatically set `OMP_NUM_THREADS` and `numactl` values

Reply:
`numactl -C` not only stands for number of cores but also index of cores, so parameter core# can only work for `OMP_NUM_THREADS`. Will try to find a simpler way to set the variables.

Reply:
always use 0-`core#-1` for numactl

Reply:
In this way, we could not specify the socket of CPU. For example, if we are going to use second socket, the command should be `numactl -C 48-95 ...`

Reply:
I don't think there is such need in bechnmarking

Reply:
Sure, then we will add parameter core# and automatically set `OMP_NUM_THREADS` and `numactl` values